docs/components/motor/gpio/encoded-motor.md

```diff
@@ -129,13 +126,15 @@ In addition to the [attributes for a non-encoded motor](/components/motor/gpio),
 | Name | Type | Inclusion | Description |
 | ---- | ---- | --------- | ----------- |
 | `encoder` | string | **Required** | `name` of the encoder. |
-| `ticks_per_rotation` | string | **Required** | Number of ticks in a full rotation of the encoder and motor shaft. |
-| `ramp_rate` | number | Optional | How fast to ramp power to motor when using RPM control. 0.01 ramps very slowly; 1 ramps instantaneously. Range is (0.0, 1.0]. <br> Default = `0.2` |
+| `ticks_per_rotation` | int | **Required** | Number of ticks in a full rotation of the encoder and motor shaft. |
+| `ramp_rate` | float | Optional | Rate to increase the motor's input voltage (power supply) per second when increasing the speed the motor rotates (RPM). <br> Range = `(0.0, 1.0]` <br> Default = `0.2` |
 
 {{% alert title="Note" color="note" %}}
 
 The attribute [`max_rpm`](/components/motor/gpio) is not required or available for encoded `gpio` motors.
 
+If `encoder` is model [`AM5-AS5048`](/components/encoder/ams-as5048/),`ticks_per_rotation` must be `1`, as `AM5-AS5048` is an absolute encoder which provides angular measurements directly.

```

Comment:
Whoa this is interesting. I’d think ticks per rotation just wouldn’t apply to an absolute encoder and shouldn’t be required. Good to know.